Makefile

```diff
@@ -25,30 +25,30 @@ MOD_FILES = util_rot.mod globalvar.mod parameters.mod \
 	boundary_rot.mod solver_rot.mod matrix_rot.mod initial_rot.mod  procedures.mod
 
 #FC = gfortran
-FC = mpif90 -O2
+#FC = mpif90 -O2
+FC = h5pfc
```

Comment:
This is breaking change for anyone using the code right now without hdf5 present. h5pfc is wrapper/helper script for compiling with hdf5. This is not ready to go into main branch as it is. At the very least need to have some kind of flag option to not use this if you want to use the current code configuration.

Reply:
Again, this is just a merge into the main on our local fork, not production.